src/PKSim.Core/Model/ExpressionProfile.cs

```diff
@@ -42,7 +42,12 @@ public override string Name
          set
          {
             if (string.Equals(Name, value))
+            {
+               //Even if the name has not changed, we raise the even here as the underlying composite name may have changed
```

Comment:
This is key to see the Node update on the right. Otherwise, because we have composite names, the name has changed already when the molecule was renamed